src/components/UI/Cards/PlayListVideoCards/PlayListVideoCards.jsx

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios';
+import React from 'react';
+import { useLocation } from 'react-router';
+import ViewCount from '../../../../utils/ViewCount';
+import IcTwotoneDelete from '../../Icons/IcTwotoneDelete';
+import "./PlayListVideoCards.css";
+
+function PlayListVideoCards({props,setPlayListVideos}) {
+    const { videoid, snippet, statistics } = props;
+    const location = useLocation();
+    const playlistId = location.search.slice(1);
+    console.log(props, videoid, snippet, statistics,playlistId)
+
+    const deletePlayListVideoFromIindividualPlayListHandler = async(id) => { 
+        try {
```

Comment:
This is wonderful.
I don't know whether it's good practice or not but it's readable that's what matters to me.

well Done.